source/_docs/configuration.markdown

```diff
@@ -23,6 +23,13 @@ Inside your configuration folder is the file `configuration.yaml`. This is the m
 
 If you run into trouble while configuring Home Assistant, have a look at the [configuration troubleshooting page](/getting-started/troubleshooting-configuration/) and at the [configuration.yaml examples](/cookbook/#example-configurationyaml).
 
+<div class='note tip'>
+
+The config files can also be accessed via the console on the Hass.io host system (which runs the homeassistant docker image)  in the `/mnt/data/supervisor/homeassistant` directory. This path can also be found by running `docker container inspect homeassistant | grep :/config`. 
+
+</div>
```

Comment:
From my point of view is this an advanced topic and should go to https://www.home-assistant.io/hassio/commandline/ as there is the console already mentioned.